tests/test_pagination.py

```diff
@@ -31,6 +32,12 @@ def participants(client):
             db.session.add(s)
         db.session.commit()
 
+        # Add study files for studies
+        for _ in range(102):
```

Comment:
Could probably create the study files and link the files to a study in the loop where you're creating each study